internal/exporter/prometheus/metrics/level.go

```diff
@@ -0,0 +1,176 @@
+// SPDX-FileCopyrightText: 2025 The Kepler Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package metrics
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Level represents the metrics level configuration using bit patterns
+type Level uint32
+
+const (
+	// MetricsLevelNone represents no metrics enabled
+	MetricsLevelNone Level = 0
+
+	// Individual metric levels using bit patterns
+	MetricsLevelNode      Level = 1 << (iota - 1) // 1
+	MetricsLevelProcess                           // 2
+	MetricsLevelContainer                         // 4
+	MetricsLevelVM                                // 8
+	MetricsLevelPod                               // 16
+
+	// MetricsLevelAll represents all metrics enabled
+	MetricsLevelAll Level = MetricsLevelNode | MetricsLevelProcess | MetricsLevelContainer | MetricsLevelVM | MetricsLevelPod
```

Comment:
Lets avoid the none and the all until there is a specific usecase. 
```
metricsLevels: [] | null
```
indicates all 


Is there a specific usecase of none ?

Reply:
Second thought, yes, we need none but not `all`